test/Common/TestHelpers.cs

```diff
@@ -48,54 +48,40 @@ internal static class TestHelpers
             IDurableHttpMessageHandlerFactory durableHttpMessageHandler = null,
             ILifeCycleNotificationHelper lifeCycleNotificationHelper = null)
         {
-            var durableTaskOptions = new DurableTaskOptions
```

Comment:
This can likely be restored to most of it's original logic. This was for the more original complicated design.